integr8ly/grafana-operator
# feat: allowing import dashboards from URL

Importing dashboards from a URL (in addition to json inside the CR)

https://issues.jboss.org/browse/INTLY-2983

Verification Steps

1) Deploy Grafana operator
2) Deploy a dashboard with URL field, there is a CR example file DashboardFromURL.yaml 
3) Access Grafana UI and verify if the dashboard was imported from URL
4) Delete the Grafana dashboard CR
5) Change/Remove the URL in the grafana dashboard CR to an invalid one
6) Deploy the dashboard again and check if it fallback to the json field
